ppl/zqd/db/db.go

```diff
@@ -35,11 +35,14 @@ type Config struct {
 }
 
 func Open(ctx context.Context, logger *zap.Logger, conf Config, root iosrc.URI) (DB, error) {
+	logger = logger.Named("db")
```

Comment:
suggest using 'database' to make reading the logs a little easier.

```suggestion
	logger = logger.Named("database")
```